projects/lights_control/src/lights_simple_peripherals.c

```diff
@@ -0,0 +1,28 @@
+#include "event_queue.h"
+#include "fsm.h"
+#include "status.h"
+
+#include "lights_events.h"
+#include "lights_simple_peripherals.h"
+
+static SimplePeripheralCallback prv_simple_callback;
```

Comment:
Should be named `s_` not `prv_` since it is a static variable. I know that is is actually a Callback but we treat that as if it were a variable.